tests/subsys/secure_storage/psa/its/testcase.yaml

```diff
@@ -0,0 +1,32 @@
+common:
+  integration_platforms:
+    - native_sim
+  platform_exclude:
+    - qemu_cortex_m0 # settings subsystem initialization fails
```

Comment:
The qemu_cortex_m0 seems broken, or at least not read data from flash.bin.
The fash0 dts definition comes from nordic  nrf51822, which causes all flash_reads on internal flash (flash0) to be done as memcpy of some memory region to target buffer, ending up with some trash in user provided buffers (this makes NVS fail), instead of using the flash.bin.